app/models/identity.rb

```diff
@@ -6,6 +6,8 @@ class Identity < ApplicationRecord
 
   delegate :metadata, to: :sp, prefix: true
 
+  CONSENT_EXPIRATION = 1.year
```

Comment:
Do we have a better home for this constant?

Reply:
I'm ok with it here.  The alternative is application.yml (.default) and making it env configurable